manim/mobject/graphing/coordinate_systems.py

```diff
@@ -1354,6 +1362,8 @@ def plot_antiderivative_graph(
             The y-value at which the graph intercepts the y-axis.
         samples
             The number of points to take the area under the graph.
+        use_vectorized
+            Whether to use the vectorized version of the antiderivative.
```

Comment:
Maybe a small one line description of what vectorised means here, or a reference to `plot` could be helpful for a first time user of this function to know what exactly this parameter does?

Reply:
Done